ksp_plugin_adapter/localization/zh-cn.cfg

```diff
@@ -200,12 +200,12 @@ Localization {
     #Principia_FlightPlan_Delete = 删除当前计划
     #Principia_FlightPlan_Rebase = 变基
     #Principia_FlightPlan_Duplicate = 重复
-    #Principia_FlightPlan_Optimization = Optimization parameters for <<1>> flyby
-    #Principia_FlightPlan_TargetAltitude = Altitude
+    #Principia_FlightPlan_Optimization = <<1>>飞略轨道参数优化
```

Comment:
I wonder if `<<1>>飞略优化参数` would be more appropriate here.

Reply:
I think `<<1>>飞掠参数优化` will be ok as well, since "parameter" only refers to orbit parameter in this context. But `优化` can't be placed in the middle of "<<1>>飞掠参数", or `优化` will be seen as a verb, and it will mean "<<1>> flyby optimizes parameters".

Reply:
Hm, that is fair. I am admittedly unfamiliar with technical terminology.

Reply:
And I'm sorry that I wrongly input "飞掠" (means flyby) as "飞略" : (
I'll fix it instantly.